src/utils/redis.ts

```diff
@@ -8,10 +8,13 @@ const clients: Redis[] = [];
  */
 export function connect(options: RedisOptions): Redis {
     // TODO: Figure out what other options we should set (timeouts, etc)
-    // TODO: pass on connection error
-    const client: Redis = new Redis(options);
-    clients.push(client);
-    return client;
+    try {
+        const client: Redis = new Redis(options); // Default port is 6379 automatically
+        clients.push(client);
+        return client;
+    } catch (err) {
```

Comment:
I don't think this will have any effect. the redis client is an event emitter and will emit an `error` event if there is a connection issue. I believe it only tries to connect then it tries to perform an operation on the redis store.

Reply:
Ok, I've removed this try-catch